core/src/main/java/org/javarosa/core/model/QuestionDef.java

```diff
@@ -48,7 +49,7 @@
  *
  * @author Daniel Kayiwa/Drew Roos
  */
-public class QuestionDef implements IFormElement, Localizable {
+public class QuestionDef extends ActionController implements IFormElement, Localizable {
```

Comment:
Can you point out where the fact that `QuestionDef` extends `ActionController` is utilized?